geowebcache/s3storage/src/test/java/org/geowebcache/s3/S3BlobStoreConfigTest.java

```diff
@@ -0,0 +1,25 @@
+package org.geowebcache.s3;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.amazonaws.services.s3.model.CannedAccessControlList;
+
+public class S3BlobStoreConfigTest {
```

Comment:
It would be good to test this with deserialization from XML.  XStream is where problems with this sort of thing are most likely to occur.